site/gatsby-site/src/components/AiidHead.js

```diff
@@ -1,14 +1,14 @@
 import React from 'react';
-import Helmet from 'react-helmet';
 import config from '../../config';
 
-const AiidHelmet = ({
+const AiidHead = ({
   children,
   metaTitle = null,
-  metaDescription = null,
+  metaDescription = metaTitle,
```

Comment:
This is a weird default. All pages should have a title and a description for SEO purposes besides just correctness.  If one of these is missing, it is okay if the build crashes.